klt-cert-manager/config/manager/manager.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     control-plane: certificate-operator
     app.kubernetes.io/name: deployment
     app.kubernetes.io/instance: certificate-operator
-    app.kubernetes.io/component: manager
+    app.kubernetes.io/component: keptn-cert-manager
```

Comment:
```suggestion
    app.kubernetes.io/component: certificate-operator
```

Reply:
this reflect also what @thisthat suggested for the metric operator so I would keep as is

Reply:
not sure I understand, revert this change ?